community/command-line/src/main/java/org/neo4j/commandline/admin/AdminTool.java

```diff
@@ -159,17 +159,22 @@ private void failure()
 
     private void failure( String message, Exception e )
     {
+        failure( message, e, 1 );
```

Comment:
It might be useful to extract `1, 2,3` as useful constants or enums to indicate what the code means.